src/NzbDrone.Core/Indexers/IndexerBase.cs

```diff
@@ -84,9 +88,16 @@ public virtual object RequestAction(string action, IDictionary<string, string> q
         protected virtual IList<ReleaseInfo> CleanupReleases(IEnumerable<ReleaseInfo> releases)
         {
             var result = releases.DistinctBy(v => v.Guid).ToList();
+            var settings = Definition.Settings as IIndexerSettings;
 
             result.ForEach(c =>
             {
+                // Use multi languages from setting if ReleaseInfo languages is empty
+                if (!c.Languages.Any() && MultiRegex.IsMatch(c.Title) && settings.MultiLanguages.Any())
```

Comment:
Might need to import the namespace, but switch to `Empty()`

```suggestion
                if (c.Languages.Empty() && MultiRegex.IsMatch(c.Title) && settings.MultiLanguages.Any())
```